src/SFA.DAS.AssessorService.Data/StandardRepository.cs

```diff
@@ -240,7 +263,7 @@ private async Task<Standard> GetLatestVersionOfAStandardByIFateReferenceNumber(s
             return result;
         }
 
-        private async Task<Standard> GetStandardsByStandardReferenceAndVersionInternal(string standardReference, string version)
+        private async Task<Standard> GetStandardByIFateReferenceNumberAndVersionInternal(string ifateReferenceNumber, string version)
         {
             var sql = @"SELECT StandardUId, IfateReferenceNumber, LarsCode, Title, Version, Level, Status, TypicalDuration, 
                             MaxFunding, IsActive, LastDateStarts, EffectiveTo, EffectiveFrom, VersionEarliestStartDate, 
```

Comment:
Column names are in square brackets on some queries and not others

Reply:
Have updated to keep it consistent